### PR TITLE
Add edit, delete person command & Update add person exception

### LINKED ISSUE #34: Create new List<Person> column in Startup and command that allows users to attach/remove associated persons to each Startup
Extension of Person/Founder Class Issue
Allow users to attach Founders/Key Employees to each startup

### PR DESCRIPTION
**Pull request by yespiggy**
* `edit-p` and `delete-p` features are added
* Duplicated person exception is now handled
* More test cases are yet to be added

### CODE REVIEW

**Review by yeoshuheng — CHANGES_REQUESTED** [review]
It seems to be failing the github CI, would it be possible to check again?

File: src/main/java/seedu/address/model/startup/Startup.java [hunk 1]
```diff
@@ -33,7 +35,19 @@ public class Startup {
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
     private List<Note> notes = new ArrayList<>();
-    private List<Person> persons = new ArrayList<>();
+    //private List<Person> persons = new ArrayList<>();
```
Comment (menatahari) [inline src/main/java/seedu/address/model/startup/Startup.java:38]: This seems to be unused. I think this can be deleted

**Review by yeoshuheng — CHANGES_REQUESTED** [review]
I think the testing for the other person-related functions aren't pushed in this pr, could you check again? :)

**Review by yeoshuheng — APPROVED** [review]
lgtm

**Pull request merged**